botfront/imports/api/instances/instances.methods.js

```diff
@@ -182,23 +265,24 @@ if (Meteor.isServer) {
                     error = new Meteor.Error('500', e);
                 }
 
-                console.log(error);
                 throw error;
             }
         },
+
+        'extractDomainFromStories'(storyGroup) {
```

Comment:
What's the purpose of this method ? It's never called from anywhere except in tests

Reply:
There was no other way of testing at the time.